packages/primitives/scripts/build.js

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
```

Comment:
Is this needed?

Reply:
This script is supposed to be an executable. Do you have any concerns here?

Reply:
If it is an executable, then we can go ahead with this.